contrib/metrics/supply_check/__init__.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import os
+import progressbar
+from slickrpc.rpc import Proxy
+
+from .deltas_mainnet import MainnetSupplyDeltas
+from .theoretical import Network, MAINNET
+
+COIN=100000000
+
+# Returns the theoretical supply, the total block rewards claimed,
+# and the block at the given height.
+#
+# - `zcashd` a slickrpc.rpc.Proxy that can be used to access zcashd
+# - `deltas` a SupplyDeltas object tracking the deltas observed
+# - `height` the block height to consider
+# - `flag` Either `1` or `2`. This flag will be provided to the
```

Comment:
Possible improvement (non-blocking): add `TXID_ONLY = 1` and `FULL_TX_DATA = 2` constants.